lms/envs/common.py

```diff
@@ -590,6 +590,18 @@
     }
 }
 
+COURSE_MODE_DEFAULTS = {
+    'bulk_sku': None,
```

Comment:
these values were originally HARD CODED on line 119 of ` common/djangoapps/course_modes/models.py`